lib/oxidized/model/ios.rb

```diff
@@ -26,15 +26,21 @@ class IOS < Oxidized::Model
   cmd :secret do |cfg|
     cfg.gsub! /^(snmp-server community).*/, '\\1 <configuration removed>'
     cfg.gsub! /^(snmp-server host \S+( vrf \S+)?( version (1|2c|3))?)\s+\S+((\s+\S*)*)\s*/, '\\1 <secret hidden> \\5'
-    cfg.gsub! /^(username \S+ privilege \d+) (\S+).*/, '\\1 <secret hidden>'
-    cfg.gsub! /^(username \S+ password \d) (\S+)/, '\\1 <secret hidden>'
-    cfg.gsub! /^(username \S+ secret \d) (\S+)/, '\\1 <secret hidden>'
-    cfg.gsub! /^(enable (password|secret) \d) (\S+)/, '\\1 <secret hidden>'
+    cfg.gsub! /^(username .+ (password|secret) \d) .+/, '\\1 <secret hidden>'
+    cfg.gsub! /^(enable (password|secret)( level \d+)? \d) .+/, '\\1 <secret hidden>'
     cfg.gsub! /^(\s+(?:password|secret)) (?:\d )?\S+/, '\\1 <secret hidden>'
     cfg.gsub! /^(.*wpa-psk ascii \d) (\S+)/, '\\1 <secret hidden>'
     cfg.gsub! /^(.*key 7) (\d.+)/, '\\1 <secret hidden>'
-    cfg.gsub! /^(tacacs-server key \d) (\S+)/, '\\1 <secret hidden>'
+    cfg.gsub! /^(tacacs-server (.+ )?key) .+/, '\\1 <secret hidden>'
```

Comment:
The "tacacs-server" command may include additional options and a key with spaces. Previously, with their presence secrets were not cut.
before:
```
tacacs-server host 192.168.100.1 nat single-connection port 5 key qqq www
```
after:
```
tacacs-server host 192.168.100.1 nat single-connection port 5 key <secret hidden>
```